lib/open_food_network/orders_and_fulfillments_report.rb

```diff
@@ -238,31 +282,55 @@ def columns
           proc { |line_items| line_items.first.order.bill_address.andand.state }
         ]
       else
-        [proc { |line_items| line_items.first.product.supplier.name },
-         proc { |line_items| line_items.first.product.name },
-         proc { |line_items| line_items.first.full_name },
-         proc { |line_items| line_items.sum(&:quantity) },
-         proc { |line_items| line_items.first.price },
-         proc { |line_items| line_items.sum { |li| li.quantity * li.price } },
-         proc { |_line_items| "" },
-         proc { |_line_items| I18n.t(:report_header_incoming_transport) }]
+        DefaultReport.new(self).columns
```

Comment:
This extract class refactoring is what I intend doing with the other types of reports soon.